### ISSUE TITLE
Is it possible to have a setting to use actual real time and location?

### ISSUE THREAD

**Issue by CATboardBETA** [dymanoid/RealTime#296]
Maybe in configuration you can chose to use specific location and you specify your timezone. For me that would be ETC/EDT depending on the time of year, so I'd group them like that rather than just having the UTC± format.

I remember in the past there being a mod that I could do this with, but maybe I'm misremembering. It was really immersive to have the sun go down in game while it was going down for me. It could also have integration with the Snowfall DLC, letting it snow more at different times of year depending on your location.

TLDR; Actual real time would be nice.
